examples/rollup/circuit_test.go

```diff
@@ -195,6 +213,7 @@ func TestCircuitUpdateAccount(t *testing.T) {
 	assert := test.NewAssert(t)
 
 	var updateAccountCircuit circuitUpdateAccount
+	(*Circuit)(&updateAccountCircuit).allocateSlicesMerkleProofs()
```

Comment:
why the cast?

Reply:
type `circuitUpdateAccount` inherits the fields but not the methods of `Circuit`